ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -224,6 +224,24 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-network-mtu-edge-zone
+- as: aws-ipi-network-mtu-localzone-sdn-f28
```

Comment:
Shall we still cover sdn configuration? cc @yunjiang29 to review.

Reply:
Confirmed with installer and network team, sdn network type is already disabled in 4.15, so we do not need to cover it in 4.15 -> 4.16 upgrade.

Reply:
Yes, SDN has been removed from >= 4.15 fresh install.